storm-core/src/ui/public/templates/component-page-template.html

```diff
@@ -211,6 +216,7 @@
         <td>{{uptime}}</td>
         <td>{{host}}</td>
         <td><a href="{{workerLogLink}}">{{port}}</a></td>
+        <td>{{processId}}
```

Comment:
Can we close the td tag?  I know it is not required but all the other ones are closed.